src/applications/static-pages/events/helpers/index.js

```diff
@@ -55,12 +55,12 @@ export const deriveMostRecentDate = (
   return futureDates[0];
 };
 
-function keepUniqueEventsFromList(allEvents, event) {
-  if (!allEvents) {
+export const addUniqueEventsToList = (allEvents, event) => {
```

Comment:
Not sure why this was named `keepUniqueEventsFromList` previously, but its new name is more appropriate for its function.